pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -143,10 +144,13 @@ func (o *ReconcileClusterRoleBindingsOptions) Validate() error {
 	return nil
 }
 
-// ReconcileClusterRoleBindingsOptions contains all the necessary functionality for the OpenShift cli reconcile-cluster-role-bindings command
 func (o *ReconcileClusterRoleBindingsOptions) RunReconcileClusterRoleBindings(cmd *cobra.Command, f *clientcmd.Factory) error {
 	changedClusterRoleBindings, err := o.ChangedClusterRoleBindings()
-	if err != nil {
+	if IsClusterRoleBindingLookupError(err) {
+		// we got a partial match, so we want to warn the user but we can continue since there's no harm in updating the
+		// bindings that we do know need to be updated for reconciliation to be complete
```

Comment:
Can we assume we're OK to use `os.Stderr` or should we pipe in error and output writers for the command?